--- buildozer.spec      2018-09-19 07:32:14.279989224 +0000
+++ buildozer.spec.default      2018-09-19 07:32:52.986992074 +0000
@@ -65,7 +65,7 @@
 #android.minapi = 8
 
 # (int) Android SDK version to use
-#android.sdk = 21
+android.sdk = 19
 
 # (str) Android NDK version to use
 #android.ndk = 9c
@@ -144,7 +144,7 @@
 [buildozer]
 
 # (int) Log level (0 = error only, 1 = info, 2 = debug (with command output))
-log_level = 1
+log_level = 2
 
 # (int) Display warning if buildozer is run as root (0 = False, 1 = True)
 warn_on_root = 1
